tools/dgeni/templates/componentGroup.template.html

```diff
@@ -79,6 +79,20 @@ <h3 id="additional_interfaces" class="docs-header-link docs-api-h3">
     {% endfor %}
   {%- endif -%}
 
+  {%- if doc.additionalFunctions.length -%}
+    <h3 id="additional_functions" class="docs-header-link docs-api-h3">
```

Comment:
Should this be separated with a dash, rather than an underscore?

Reply:
Don't think so. I've just made it consistent with the other header links.